clarify admin access to sub account for sso (#562)
logzio/logz-docs: _source/user-guide/users/single-sign-on/index.md

Permissions are set at the group level
unless a group is set to "Configured per user".
If a user belongs to multiple groups,
they will get the highest permissions set.

For example:
If someone is part of an "User" group and an "Admin" group,
they'll receive admin permissions.

###### Admin access to sub accounts

You'll need to ensure you have at least one Admin user explicitely set for every sub account before enabling SSO.

> Once SSO is enabeled, Admins on the main account will not be have admin access to sub accounts unless they are already configured as admins on the sub account before SSO was enabled.
> 

Author: ```suggestion
Once SSO is enabeled, Admins on the main account will not have admin access to sub accounts unless they were already configured as admins on the sub account before SSO was enabled.

```
